### PR TITLE
Add 1.19.0 release Blog && update latest version

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
* **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Which issue(s) this PR fixes**:

Fixes #

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Add 1.19.0 release Blog && update latest version

https://github.com/kubeedge/kubeedge/issues/5927

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: blog/release-v1.19/index.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+authors:
+- KubeEdge SIG Release
+categories:
+- General
+- Announcements
+date: 2024-10-28
+draft: false
+lastmod: 2024-10-28
+summary: KubeEdge v1.18 is live!
```
Comment (wbc6080) [inline blog/release-v1.19/index.mdx:10]:
```suggestion
summary: KubeEdge v1.19 is live!
```

Comment (Shelley-BaoYue) [reply blog/release-v1.19/index.mdx]: > 

Fixed. Thx

**Review by fisherxu — COMMENTED** [review]
/approve

**Review by wbc6080 — COMMENTED** [review]
/lgtm

**Pull request merged**